# Total planning attempts
max_solutions: 5

# Planning group and link names
arm_group_name: "arm"
eef_name: "end_effector"
hand_group_name: "hand"
hand_frame: "link_art6"

# Poses
hand_open_pose: "opened"
hand_close_pose: "closed"
arm_home_pose: "vertical"

# Scene frames
world_frame: "dummy_link"
table_reference_frame: "dummy_link"
object_reference_frame: "dummy_link"
surface_link1: "counter"
surface_link2: "stove"

# Collision object for picking
# CYLINDER object specifications
object_name: "saucepan"
object_dimensions: [0.25, 0.02] # [height, radius]
object_pose: [0, 0.3, 0.085, 0, 0, 0.785]
object_scale: 0.001

# Gripper grasp frame transform [x,y,z,r,p,y]
grasp_frame_transform: [0, 0, 0.17, 3.14, 0, 0]

# Place pose [x,y,z,r,p,y]
place_pose: [0.15, 0.3, 0.095, 0, 0, 0]
place_surface_offset: 0.0001 # place offset from table

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0
approach_object_max_dist: 0.2

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.01
lift_object_max_dist: 0.1